vllm/envs.py

```diff
@@ -116,6 +116,7 @@
     VLLM_NIXL_SIDE_CHANNEL_HOST: str = "localhost"
     VLLM_NIXL_SIDE_CHANNEL_PORT: int = 5557
     VLLM_ALL2ALL_BACKEND: str = "naive"
+    VLLM_MODELOPT_MAX_TOKENS_PER_EXPERT: int = 163840
```

Comment:
I think we should use the kernel name rather than `MODELOPT` since this is not specific to the `MODELOPT` integration and these kernels will be leveraged by multiple integrations

Reply:
Changed it to: `VLLM_MAX_TOKENS_PER_EXPERT_BLOCKSCALE`